core/src/main/java/com/gentics/mesh/core/endpoint/node/BinaryDownloadHandler.java

```diff
@@ -5,9 +5,15 @@
 import static io.netty.handler.codec.http.HttpResponseStatus.BAD_REQUEST;
 import static io.netty.handler.codec.http.HttpResponseStatus.NOT_FOUND;
 
+import java.util.Objects;
+import java.util.function.Predicate;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
 import com.gentics.mesh.context.InternalActionContext;
 import com.gentics.mesh.context.impl.InternalRoutingActionContextImpl;
-import com.gentics.mesh.core.data.HibBinaryDataElement;
+import com.gentics.mesh.core.data.HibAntivirableBinaryElement;
```

Comment:
The antivirus-checkable binary entity has been split away from the ordinary binary data definition - image variants are not necessary to check.